pkg/apis/servicecatalog/types.go

```diff
@@ -282,30 +282,30 @@ type ServiceClassStatus struct {
 	PresentInCatalog bool
 }
 
-// ServicePlanList is a list of ServicePlans.
-type ServicePlanList struct {
+// ClusterServicePlanList is a list of ServicePlans.
+type ClusterServicePlanList struct {
 	metav1.TypeMeta
 	metav1.ListMeta
 
-	Items []ServicePlan
+	Items []ClusterServicePlan
 }
 
 // +genclient=true
 // +nonNamespaced=true
 
-// ServicePlan represents a tier of a ServiceClass.
-type ServicePlan struct {
+// ClusterServicePlan represents a tier of a ClusterServiceClass.
+type ClusterServicePlan struct {
 	metav1.TypeMeta
 	metav1.ObjectMeta
 
-	Spec   ServicePlanSpec
+	Spec   ClusterServicePlanSpec
 	Status ServicePlanStatus
```

Comment:
Status needs to be renamed

Reply:
We left the BrokerStatus as was as I was not thinking the status would necessarily be specific to Cluster level resource. So for consistency, didn't rename it here either.